modules/hypothesis.py

```diff
@@ -15,20 +17,75 @@ def get_items(self) -> Mirror.Results:
                 continue
             yield node(
                 heading=dt_heading(page.created, link(title=page.title, url=page.url)),
-                children=[node(
-                    heading=dt_heading(hl.created, link(title='context', url=hl.hyp_link)),
-                    tags=hl.tags,
-                    body=hl.highlight,
-                    children=[] if hl.annotation is None else [node(
-                        heading=hl.annotation,
-                    )]
-                ) for hl in page.highlights]
+                properties=self.get_page_properties(page),
+                children=[
+                    node(
+                        heading=dt_heading(
+                            hl.created, link(title="context", url=hl.hyp_link)
+                        ),
+                        properties=self.get_annotation_properties(hl),
+                        tags=hl.tags,
+                        body=hl.highlight,
+                        children=[]
+                        if hl.annotation is None
+                        else [
+                            node(
+                                heading=self.get_annotation(hl.annotation).split("\n")[
+                                    0
+                                ],
+                                body=self.get_annotation(hl.annotation),
```

Comment:
Hmm, kind of annoying that we'll call `.get_annotation` twice, in particular because this would cause pandoc conversion twice (so might be particularly slow). Maybe extract a lambda that constructs such `node` by splitting the first line?